app/graphql/types/enum/favorite_item.rb

```diff
@@ -0,0 +1,6 @@
+class Types::Enum::FavoriteItem < Types::Enum::Base
```

Comment:
Hmmm, could you rename this to `FavoriteType`? I think it would better express what this is, and allow us to use it elsewhere (ie, in queries like `user { favorites(type: ANIME) }`